client/van_router_create.go

```diff
@@ -181,8 +181,12 @@ func (cli *VanClient) GetVanRouterSpecFromOpts(options types.VanSiteConfigSpec,
 	} else {
 		van.Name = options.SkupperName
 	}
+	if options.SkupperNamespace == "" {
+		van.Namespace = cli.Namespace
```

Comment:
So now cli.Namespace, is a kind of default? is it possible to just remove it?

Reply:
With this patch, the cli namespace and the namespace for site deployment can be different. Previously, it assumed they were the same namespace (e.g. default). The cli has not been updated to enable deployment to a different namespace but we could make that change in a future patch.

Reply:
The cli already allows the namespace to be specified, and the cli namespace is then set accordingly. I don't see any reason for wanting to then ignore that and specify a namespace via some alternative option.

Reply:
The cmd/skupper/skupper.go ensures that the cli namespace setting is defined as originally intended in the create opts. This was just a check to ensure that the options.SkupperNamespace is defined as it is now required and was defaulting to the cli.Namespace.  In the case of the site-controller, it is set to the site namespace.

Alternatively, we could treat an empty SkupperNamespace here as an error/exception if it is not spec'ed in the options.